core/src/actors/storage_manager.rs

```diff
@@ -67,10 +73,41 @@ impl Actor for StorageManager {
 
     /// Method to be executed when the actor is started
     fn started(&mut self, ctx: &mut Self::Context) {
-        // Stop context if the storage is not properly initialized
-        if self.storage.is_none() {
-            ctx.stop();
-        }
+        debug!("Storage Manager actor has been started!");
+
+        // Get address to launch the server
+        let config_manager_addr = System::current().registry().get::<ConfigManager>();
+
+        // Start chain of actions
+        config_manager_addr
+            // Send GetConfig message to config manager actor
+            // This returns a Request Future, representing an asynchronous message sending process
+            .send(GetConfig)
+            // Convert a normal future into an ActorFuture
+            .into_actor(self)
+            // Process the response from the config manager
+            // This returns a FutureResult containing the socket address if present
+            .then(|res, _act, _ctx| {
+                // Process the response from config manager
+                process_get_config_response(res)
+            })
+            // Process the received config
+            // This returns a FutureResult containing a success or error
+            .and_then(|config, act, ctx| {
+                let db_path = config.storage.db_path;
+
+                *act = Self::new(&db_path.to_string_lossy());
+
+                // Stop context if the storage is not properly initialized
+                // TODO check error handling
```

Comment:
Thanks for adding the `TODO` reference! There is already an issue (#72) talking about the error handling we should be defining anytime soon. Please change this TODO to a FIXME addressing this issue, and please update the issue description as well to include this case. 

```suggestion
                // FIXME(#72): check error handling
```